pages/common/mix.md

```diff
@@ -0,0 +1,27 @@
+# mix
+
+> Mix is a build tool that provides tasks for creating, compiling, and testing Elixir projects, managing its dependencies, and more.
+
+- List all mix commands:
+
+`mix help`
+
+- Execute particular file:
+
+`mix run {{my_script.exs}}`
+
+- Create a new project:
+
+`mix new {{project_name}}`
+
+- Compile project:
+
+`mix compile`
+
+- Start an iex session inside the project:
```

Comment:
This looks like a different command. Perhaps it should have its own page? We can always add a second line to the description saying `See also`.

Reply:
On second thoughts, if this is a very integral part of the workflow. We can keep it and put iex within backticks. This would be something like the locate.md page, where the `updatedb` command is in the same page because it is synonymous with `locate`. But only if it is used very regularly along with `mix`, otherwise this should move to a separate page.